dashboard/src/components/InstanceListView/index.tsx

```diff
@@ -15,6 +16,7 @@ import {
 } from "../ErrorAlert";
 import PageHeader from "../PageHeader";
 import SearchFilter from "../SearchFilter";
+import AlphaWarning from "./AlphaWarning";
```

Comment:
Instead of creating another warning message I'd try to reuse the generic ones that we already have.